### PR TITLE
dev guide 0.6.0

### PR DESCRIPTION
**Pull request by maelle**
Trying something new: for review the links are to the dev dev guide, I'll switch to production dev guide links after the release of said dev guide!

### CODE REVIEW

File: content/blog/2021-02-23-devguide-0.6.0/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+> We have resumed activity after a break due to the COVID-19 crisis, but are being gentle with deadlines and timing, giving a lot of grace to all people involved. We're all doing our best and rOpenSci is trying to be accommodating with schedules during this challenging period.
+
+## Now in Spanish! ¡Ahora en español!
+
+We're extending our software review system to submissions in Spanish, as [we had hinted to in a forum post](https://discuss.ropensci.org/t/language-documentation-for-a-package/2221/3) and other venues!
+The first package to be reviewed in Spanish is [
+censo2017](https://github.com/ropensci/software-review/issues/414) by [Pachá (aka Mauricio Vargas Sepúlveda)](/author/pachá-aka-mauricio-vargas-sepúlveda/) that provides access to the 2017 Chilean Census.
+Editor [Melina Vidoni](/blog/2019/01/31/more_editors/) is in charge of this review, the reviewers are [Frans van Dunné](https://github.com/FvD) and [María Paula Caldas](https://www.mpaulacaldas.com/).
+Frans van Dunné most kindly translated our [review template to Spanish](https://devdevguide.ropensci.org/reviewtemplatees.html)!
```
Comment (steffilazerte) [inline content/blog/2021-02-23-devguide-0.6.0/index.Rmd:38]: I think there's a double "dev" here, and I assume this link will work after Dev Guide 0.6.0 is published?
```suggestion
Frans van Dunné most kindly translated our [review template to Spanish](https://devguide.ropensci.org/reviewtemplatees.html)!
```

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle, as always, looks great :grin: One comment about the link to the dev guide, but other than that, we're good to go! I'll probably come up with a tweet draft later next week, but feel free to post your own if the mood strikes you 😸 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

**Comment by steffilazerte** [comment]
One question, I didn't realize until now, when you say now in Spanish, do you mean the guide has a Spanish translation or only the review process and template? Perhaps that should be clarified, to "On our way to Spanish!" or something like that? Less catchy, I know 😦

**Comment by maelle** [comment]
Thanks for your attention to detail @steffilazerte :sparkles:

**Comment by steffilazerte** [comment]
Great! Draft tweet below

---

[blog] rOpenSci Dev Guide 0.6.0: Updates 📘

🎉 From @ma_salmon and the dev guide team

- Spanish software review now available!
- Guidance for docs, testing, and 📦 evolution
- Design refresh and better citation info

https://ropensci.org/blog/2021/02/23/devguide-0.6.0/

#rstats #SoftwarePeerReview

(1/2)

---

🙏 Thank yous go out to 

@fransvandunne for translating the peer review checklist into Spanish and @grusonh for helping to clarify citation information. 
And to all past and current contributors!

(2/2)

**Comment by maelle** [comment]
Awesome! 

"From @ma_salmon and the dev guide team" -> "From @ma_salmon and all software review editors".

**Comment by maelle** [comment]
Thank you!

**Pull request merged**